man-roxygen/DbmsDetails.R

```diff
@@ -15,7 +15,10 @@
 #' @param password           The password for that user.
 #' @param server             The name of the server.
 #' @param port               (optional) The port on the server to connect to.
-#' @param schema             (optional) The name of the schema to connect to.
+#' @param schema             (DEPRECATED. optional) The name of the schema to connect to. 
+#'                           DatabaseConnector attempts to set the schema using USE statement, but
+#'                           not all DBMS support USE statements. Instead use @@databaseSchema in
```

Comment:
`@databaseSchema` is a SqlRender convention (actually, not even that, an OHDSI conventions), so many people may not know what is meant by this. Try rephrasing (see previous comment).